powertools-common/pom.xml

```diff
@@ -27,10 +27,8 @@
         <version>2.0.0-SNAPSHOT</version>
     </parent>
 
-    <name>Powertools for AWS Lambda (Java) library Core</name>
-    <description>
-        A suite of utilities for AWS Lambda Functions that makes tracing with AWS X-Ray, structured logging and creating custom metrics asynchronously easier.
-    </description>
+    <name>Powertools for AWS Lambda (Java) library Common Internal Utilities</name>
```

Comment:
@jeromevdl how about this (and description)

Reply:
I would change description to something like

> Internal utilities used in Lambda Powertools modules, do not use it directly in your project

Reply:
Done